src/app/project/view-projects-logic/view-projects-logic.component.html

```diff
@@ -33,13 +33,17 @@
 
       <!-- Status Column -->
       <ng-container matColumnDef="status">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <!--<p> added by serge-->
```

Comment:
Is there any reason for this to stay in the source code?  If not, we should remove it and keep the code clean.